cl_sii/extras/dj_form_fields.py

```diff
@@ -0,0 +1,72 @@
+"""
+cl_sii "extras" / Django form fields.
+
+"""
+try:
+    import django
+except ImportError as exc:  # pragma: no cover
+    raise ImportError("Package 'Django' is required to use this module.") from exc
+
+from typing import Any, Optional
+
+import django.core.exceptions
+import django.db.models
+import django.db.models.fields
+import django.forms
+from django.utils.translation import gettext_lazy as _
+
+from cl_sii.rut import Rut
+
+
+class RutField(django.forms.CharField):
+
+    """
+    Django form field for RUT.
+
+    * Python data type: :class:`cl_sii.rut.Rut`
+
+    .. seealso::
+        :class:`.dj_model_fields.RutField`
+
+    """
+
+    default_error_messages = {
+        'invalid': _('Enter a valid RUT.'),
+    }
+
+    def __init__(self, *, validated_dv: bool = False, **kwargs: Any) -> None:
```

Comment:
why not an invalid DV message like in #5 ?

Reply:
@yaselc ?

Reply:
@glarrain do you mean using a separate error message for the case where the DV is not valid? e.g.

```python
default_error_messages = {
        'invalid': _('Enter a valid RUT.'),
        'invalid_dv': _('RUT\'s "digito verificador" is incorrect.')
    }
```

Reply:
@glarrain? 
@jtrh can you help shed some light on this issue?

Reply:
> @glarrain do you mean using a separate error message for the case where the DV is not valid?

I think that is what @glarrain meant.

You would need to change the `ValidationError` message in https://github.com/fyntex/lib-cl-sii-python/pull/197/files#diff-f6d12abbd698f974b0356078c32cbfcdb4b570857bf409850df17b2475b8cec0R56-R60 and differentiate between different error causes in https://github.com/fyntex/lib-cl-sii-python/pull/197/files#diff-f6d12abbd698f974b0356078c32cbfcdb4b570857bf409850df17b2475b8cec0R62-R68.